documented how to manually restart PostgreSQL (#2605)
CrunchyData/postgres-operator: docs/content/tutorial/administrative-tasks.md

---
title: "Administrative Tasks"
date:
draft: false
weight: 105
---

## Manually Restarting PostgreSQL

> There are times when you might need to manually restart PostgreSQL. This can be done, with minimal downtime, by using PGO's rolling update functionality. By simply adding or updating a custom annotation to the cluster instance-set's `spec.metadata.annotations` section, the Pod template will change for all instances and trigger a rolling update for the cluster. 

Reviewer: 🤔 Over in the [HA section](https://github.com/CrunchyData/postgres-operator/blob/master/docs/content/architecture/high-availability.md#rolling-updates), we seemed to introduce the idea of "rolling update" then transition to calling it "rolling restart". I wonder if we should use that language here?

Something like this?
```suggestion
There are times when you might need to manually restart PostgreSQL. This can be done by adding or updating a custom annotation to the cluster's `spec.metadata.annotations` section. PGO will notice the change and perform a [rolling restart]({{< relref "/architecture/high-availability.md" >}}#rolling-update).
```

> For example, if I have a cluster named `hippo` in the namespace `postgres-operator`, all you need to do is patch the hippo postgrescluster with the following:

Reviewer: ```suggestion
For example, if you have a cluster named `hippo` in the namespace `postgres-operator`, all you need to do is patch the hippo postgrescluster with the following:
```
